src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -432,10 +433,12 @@ protected function dateBasedWhere($type, Builder $query, $where)
      */
     protected function wrapValue($value)
     {
-        if ($this->isReserved($value)) {
-            return parent::wrapValue($value);
+        if ($value === '*') {
+            return $value;
         }
 
-        return $value !== '*' ? sprintf($this->wrapper, $value) : $value;
+        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+
+        return '"' . str_replace('"', '""', $value) . '"';
```

Comment:
I don't think replacing `"` with `""` will work with Oracle, will it? From what I've read, Oracle does not allow double quotes in identifiers whatsoever.

> Quoted identifiers can contain any characters and punctuations marks as well as spaces. However, neither quoted nor nonquoted identifiers can contain double quotation marks or the null character (\0). - [Oracle docs](https://docs.oracle.com/cd/B28359_01/server.111/b28286/sql_elements008.htm)